Tests/StreamChatUITests/SnapshotTests/VoiceRecording/Views/LiveRecordingView_Tests.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
```

Comment:
It looks like the current time indicator can be shorter than the slider height. Is that expected?

Reply:
This is expected as you can also see on the snapshot from Design's Figma
![image](https://github.com/GetStream/stream-chat-swift/assets/472467/f3466f70-35ea-45ab-af48-9648835fb887)